src/shadowbox/server/manager_service.ts

```diff
@@ -127,6 +127,8 @@ export function bindService(
     service.setPortForNewAccessKeys.bind(service)
   );
 
+  apiServer.get(`${apiPrefix}/access-keys/:id`, service.getAccessKey.bind(service));
+
   apiServer.post(`${apiPrefix}/access-keys`, service.createNewAccessKey.bind(service));
   apiServer.get(`${apiPrefix}/access-keys`, service.listAccessKeys.bind(service));
 
```

Comment:
Let's move the get key to after the list keys, so all the ` access-keys/:id` entries are together.

```suggestion
  apiServer.post(`${apiPrefix}/access-keys`, service.createNewAccessKey.bind(service));
  apiServer.get(`${apiPrefix}/access-keys`, service.listAccessKeys.bind(service));

  apiServer.get(`${apiPrefix}/access-keys/:id`, service.getAccessKey.bind(service));
```